yarn.lock

```diff
@@ -1061,6 +1061,7 @@ antigravity@artsy/antigravity:
   version "0.1.3"
   resolved "https://codeload.github.com/artsy/antigravity/tar.gz/a4438d2fe9d0cdf71f1c47faba371cd3d004e140"
   dependencies:
+    coffeescript "1.11.1"
```

Comment:
🤔

Reply:
Yeah, older artsy dependencies shipped coffescript code, not compiled javascript - I've updated one of two of these, but not all